scripts/beachball/index.ts

```diff
@@ -15,4 +21,24 @@ export const config: BeachballConfig = {
     },
     groups: [getVNextChangelogGroups()],
   },
+  hooks: {
+    postpublish: async (packagePath, name, version) => {
+      await writeFile(path.resolve(packagePath, '.npmrc'), '//registry.npmjs.org/:_authToken=${NPM_TOKEN}');
+
+      // Following https://github.com/microsoft/fluentui/pull/20352 all prerelease versions are released with `latest`
+      // Adds a post publish hook to continue to tag prerelease versions with appropriate tags
+      // can be removed once v9 is fully released
+      const tag = 'beta';
+      if (version.includes(tag) && process.env.RELEASE_VNEXT) {
+        console.log(`tagging ${name}@${version} with ${tag}`);
+        const res = await exec(`npm dist-tag add ${name}@${version} ${tag}`);
```

Comment:
I wonder if it would be better to just spawnSync with stdio: inherit and check the exit code?

Reply:
And regardless of specific exec/spawn method you should probably check the exit code instead of stderr

Reply:
in commit [7b837da](https://github.com/microsoft/fluentui/pull/20471/commits/7b837dafe3007b46a8816b23b74f15c61f837524) I ended up manually promisifying and checking the status code of `exec`

Reply:
I don't think the promise approach is really necessary (since this won't be done multi-threaded)? Not super important though.